client/src/api/api_client.ts

```diff
@@ -4,22 +4,46 @@ import axios from 'axios';
  * Client to interface with API.
  */
 class ApiClient {
-  static API_URL = 'https://ei2tz84crb.execute-api.us-east-2.amazonaws.com/dev/geolocate/';
+  static API_URL = 'https://ei2tz84crb.execute-api.us-east-2.amazonaws.com/dev';
   /**
    *   Retrieve geo ids of the geographic entities which intersect with the
    *   lat, long pair.
    */
   getGeoIdsFromLatLong = async (lat: string, long: string): Promise<ApiResponse> => {
```

Comment:
nit: I know this wasn't added here, but could we make it just getGeoIds (consistent with getWaterSystem and generally I think avoid putting param names in function names since they are self explanatory)

Reply:
Yeah good point thanks!